src/main.js

```diff
@@ -39,7 +39,7 @@ const template = [{
 function createWindow() {
     debug('creating window');
     // Create the browser window.
-    mainWindow = new BrowserWindow({ width: 1500, height: 768, resizable: true, minWidth: 1366, minHeight: 768 });
+    mainWindow = new BrowserWindow({ width: 1500, height: 768, resizable: true, minWidth: 1366, minHeight: 768, webPreferences: { webSecurity: false } });
```

Comment:
Since this issue only affects local dev, and it reduces security, can we replace this with `{ webSecurity: process.env.NODE_ENV !== 'production' }`?

Reply:
Hmm actually I know we set that environment variable when we package the application but I have no idea if that actually gets reflected into the program - can you please investigate and let me know if this works?

Reply:
nice, this works!

Reply:
What is `process.env.NODE_ENV` set to when running for real (ie, when running the packaged version that the customer will run)?

Reply:
it gets set to "production" when we do the build, so i'm guessing that carries over and is set whenever a user runs the executable?

Reply:
I think you should confirm that guess as I'm not sure that's the case.

Reply:
Whatever it is, I'm sure it isn't `development` - which is why I inverted the conditional I originally suggested. If it's unset then `null !== 'development'` will be true and security is enabled

Reply:
i'll do a build and confirm it just to be sure

Reply:
<img width="291" alt="Screen Shot 2019-03-20 at 3 20 55 PM" src="https://user-images.githubusercontent.com/43550949/54723132-ce625480-4b23-11e9-83ba-7d534f3190f9.png">
works in prod

Reply:
> <img alt="Screen Shot 2019-03-20 at 3 20 55 PM" width="291" src="https://user-images.githubusercontent.com/43550949/54723132-ce625480-4b23-11e9-83ba-7d534f3190f9.png">
> works in prod

You should run `unset NODE_ENV` before running WRW. I'd be surprised if the value of the environment variable at build time is baked into the binary.

Reply:
where should i run unset? just in any terminal on my machine?

Reply:
in the terminal you are going to use to start wrw.

Although, it seems like there are better ways to check if the app is completely packaged or not: * https://github.com/sindresorhus/electron-is-dev/blob/master/index.js
* https://github.com/electron-userland/electron-builder/issues/1966

One other thing I'm not quite clear on is this: does the end user need to make these cross origin requests? Or are they only made during WRW development?

Reply:
CORS is only an issue for local dev. During local dev, the WRW is hosted on a local webserver in order to live-reload. When packaged it is compiled to a static site that is opened by Electron. So no server == no origin == no cross-origin requests

Reply:
Thanks for the clarification.